x-pack/plugins/cases/common/api/cases/comment/index.ts

```diff
@@ -185,7 +185,7 @@ export const CommentRequestRt = rt.union([
   PersistableStateAttachmentRt,
 ]);
 
-export const CommentResponseRt = rt.intersection([
+export const CommentRt = rt.intersection([
```

Comment:
Should we use the new terminology and rename it to `AttachmentRt`? I am not sure 🙂.